save-preprocessor/src/test/kotlin/org/cqfn/save/preprocessor/DownloadProjectTest.kt

```diff
@@ -121,11 +134,78 @@ class DownloadProjectTest(
         assertions.orTimeout(60, TimeUnit.SECONDS).join().forEach { Assertions.assertNotNull(it) }
     }
 
+    @Suppress("TOO_LONG_FUNCTION")
+    @Test
+    fun testSaveProjectAsBinaryFile() {
+        val binFile = File(binFilePath)
```

Comment:
Please check in this test that if binFile is not empty, then it is successfully saved as well

Reply:
I updated test to check it